### PR TITLE
doc: update acrn ootb doc

### PR DESCRIPTION
**Pull request by leilu**
1. Update acrn.efi name in acrn ootb scripts
2. Add licenses to acrn ootb scripts
3. Add format U disk step
4. Add execute permission to ootb script
5. Fix wrong img name

Signed-off-by: Lei, Lu <leix.lu@intel.com>

### CODE REVIEW

File: doc/tutorials/acrn_ootb.rst [hunk 1]
```diff
@@ -132,6 +134,12 @@ Step 1: Create a Service VM YAML file and script
 
         exit 0
 
+     Add execute permission to the script:
```
Comment (deb-intel) [inline doc/tutorials/acrn_ootb.rst]:
```suggestion
     Grant execute permission to the script:
```

File: doc/tutorials/acrn_ootb.rst [hunk 2]
```diff
@@ -241,6 +252,12 @@ Step 1: Create a Service VM YAML file and script
...
```
Comment (deb-intel) [inline doc/tutorials/acrn_ootb.rst]:
```suggestion
     Grant execute permission to the script:
```

File: doc/tutorials/acrn_ootb.rst [hunk 3]
```diff
@@ -265,14 +282,41 @@ Use the clr-installer to build the Service VM image.
...
-  The ``service-os-industry.img`` will be generated at current directory.
+  The ``sos-industry.img`` will be generated at current directory.
 
 .. _deploy_ootb_service_vm:
 
 Step 3: Deploy the Service VM image
 ===================================
 
 #. Prepare a U disk with at least 8GB memory.
+#. Format the U disk
```
Comment (deb-intel) [inline doc/tutorials/acrn_ootb.rst]:
```suggestion
#. Prepare a U disk with at least 8GB memory. Begin by formatting the U disk:
```

Comment (deb-intel) [reply doc/tutorials/acrn_ootb.rst]: The first step is not really a step, is it? It does not contain any instructions. Perhaps we should combine #1 and #2 as shown above.

File: doc/tutorials/acrn_ootb.rst (hunk 1 above)
Comment (leilu) [reply doc/tutorials/acrn_ootb.rst]: Fixed, thanks a lot.

File: doc/tutorials/acrn_ootb.rst (hunk 3 above)
Comment (leilu) [reply doc/tutorials/acrn_ootb.rst]: Fixed, thanks a lot.

**Pull request merged**